content/cloud/quickstart.md

```diff
@@ -34,7 +34,15 @@ First, you need to have Spin installed on your computer. Use the command below t
 curl -fsSL https://developer.fermyon.com/downloads/install.sh | bash
 ```
 
-This command downloads and unpacks the latest Spin binary in the current directory. You can now run spin using the command `./spin`.
+This command downloads and unpacks the latest Spin binary in the current directory.
+
+**It's highly recommended to add Spin to a folder, which is on your path**, e.g.:
```

Comment:
I don't know about _highly_; it works fine if it isn't in your PATH (as long as you remember where it _is_). Also suggest a little rewording:

```suggestion
**It's recommended to add Spin to a directory in your `$PATH`**, e.g.:
```

Reply:
Thanks @lann